### PR TITLE
data: Webdevava

### PR DESCRIPTION
**Pull request by Webdevava**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6615"></a>

### CODE REVIEW

File: data/Webdevava.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Ankur Auti",
+  "displayStatsPublic": true,
```
Comment (khushalarora) [inline data/Webdevava.json:3]: You can remove this line, it is not required anymore :+1: 
```Suggestion
```

File: data/Webdevava.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "personal",
+  "bio": "I'm Ankur, a MERN stack developer building scalable web applications. Skilled in MongoDB, Express.js, React.js, and Node.js, and proficient in Git, npm, and Webpack. Passionate about collaborating with others to bring ideas to life and always eager to learn new technologies.",
+  "avatar": "https://github.com/Webdevava.png",
```
Comment (khushalarora) [inline data/Webdevava.json:6]: You can remove this line, it is not required anymore.
Thankyou :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @Webdevava, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**